atom/browser/atom_web_ui_controller_factory.cc

```diff
@@ -52,9 +53,16 @@ AtomWebUIControllerFactory::CreateWebUIControllerForURL(content::WebUI* web_ui,
     base::StringPairs toplevel_params;
     base::SplitStringIntoKeyValuePairs(url.query(), '=', '&', &toplevel_params);
     std::string stream_id, src;
+
+    const net::UnescapeRule::Type unescape_rules =
```

Comment:
`SPOOFING_AND_CONTROL_CHARS` should be avoided. Also `NORMAL` rule need not be specified, any rule specified includes the normal rule by default.